opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -50,9 +49,26 @@ namespace Opm {
             }
         }
 
-        /// initPORV uses doubleGridProperties: PORV, PORO, NTG, MULTPV
+        // this funcion applies a single pore volume multiplier (i.e., it deals with a single
+        // record of the MULTREGP keyword)
+        void applyPorosityRegionMultiplier_(std::vector<double>& porev,
+                                            const std::vector<int>& regionIdx,
```

Comment:
Minor nitpick: Can we have e.g. `regionValue` (or `regionId`) instead of `regionIdx`? The point is that the values in the region field are not necessarily zero offset - and can also have "holes" in the sequence of values.